lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -58,6 +58,13 @@ clear_input_value:
       id: <clear_field_id>
     op: clear
     type: input
+clear_value:
```

Comment:
Why not change `clear_input_value` directly? Also I think it's ok to just add clear operation into `set_input_value` since when we want to set a value into input, we must clear it first